.github/workflows/tests.yml

```
@@ -22,8 +22,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04]
-        pyv: ["3.8"]
+        os: [ubuntu-22.04]
+        pyv: ["3.12"]
```

I am not an expert in devops, but maybe we want to run tests on older Python versions as well? py39 [is still being supported](https://devguide.python.org/versions/).
```suggestion
        pyv: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

---

That would make 5x3=15 CI runs. Probably we should test all those python versions with "earliest", but only 3.12 for "latest" and "dev".